libs/blocks/youtube/youtube.js

```diff
@@ -14,12 +14,19 @@ export default function init(a) {
         webkitallowfullscreen mozallowfullscreen allowfullscreen
         allow="encrypted-media; accelerometer; gyroscope; picture-in-picture"
         scrolling="no"
+        id='player-${id}'
         title="${title}">
       </iframe>
     </div>`;
     a.insertAdjacentHTML('afterend', embedHTML);
     a.remove();
   };
-
   createIntersectionObserver({ el: a, callback: embedVideo });
+  document.onreadystatechange = () => {
```

Comment:
Should we not move this within the `embedVideo`? 
Otherwise, this might get tracked while it is never shown on the page.

Reply:
moved it up